index.js

```diff
@@ -3,8 +3,7 @@ const path = require('path');
 const fs = require('fs-extra');
 const opta = require('opta');
 const parseList = require('safe-parse-list');
-const { promisify } = require('util');
-const readPkg = promisify(require('read-package-json'));
+const PackageJson = require('@npmcli/package-json');
```

Comment:
```suggestion
const { create, load } = require('@npmcli/package-json');
```

Reply:
Do you think this is a style choice? Or is there a benefit to this approach? Just want to make sure I understand.

Reply:
both - the benefit is that it's easier at a glance to see what code paths are used in the package, and it's much less likely there'll be a typo trying to navigate off of `PackageJson`, and it's robust against someone mutating the PackageJson object later.